src/libstd/keyword_docs.rs

```diff
@@ -26,3 +26,35 @@
 ///
 /// [book]: https://doc.rust-lang.org/book/second-edition/ch03-03-how-functions-work.html
 mod fn_keyword { }
+
+#[doc(keyword = "let")]
+//
+/// The `let` keyword.
+///
+/// The `let` keyword is used to declare a variable.
+///
+/// Example:
+///
+/// ```rust
+/// # #![allow(unused_assignments)] fn main() {
+/// let x = 3; // We create a variable named `x` with the value `3`.
+/// # }
+/// ```
+///
+/// By default, all variables are **not** mutable. If you want a mutable variable,
+/// you'll have to use the `mut` keyword.
+///
+/// Example:
+///
+/// ```rust
+/// # #![allow(unused_assignments)] fn main() {
+/// let mut x = 3; // We create a mutable variable named `x` with the value `3`.
```

Comment:
You don't have to include a `fn main` to apply these crate-level lints. Rustdoc will already move them outside its generated `main` function.

Reply:
Oh cool! Completely forgot about it. :)